examples/api/lib/sample_templates/cupertino.0.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This is a template file for illustrating best practices when creating a
+// Flutter API example that uses the Cupertino library. To use it, copy the file
+// to your destination filename, search/replace 'Placeholder' with the name of
+// the class this is an example for, delete this block, and implement your
+// example.
+//
+// The name and location of this file should be:
+//
+//   examples/api/lib/<library>/<filename_without_dart>/<lower_snake_symbol>.<index>.dart
+//
+// So, if your example is the third example of the Foo.bar function in the
+// "baz.dart" file in the cupertino library, then the filename for your example
+// should be:
+//
+//   examples/api/lib/cupertino/baz/foo_bar.2.dart
+//
+// and it's associated test should be in:
+//
+//   examples/api/test/cupertino/baz/foo_bar.2_test.dart
+//
+// The following doc comment should remain, and be a doc comment so that the
+// symbol will be linked in the IDE. Don't put the whole description of the
+// example, since that should already be in the API docs where this example is
+// referenced, and we don't want the two descriptions to diverge. If this sample
+// is referenced more than once, link back to the instance the example file is
+// named for.
+
+/// Flutter code sample for [Placeholder].
+
+import 'package:flutter/cupertino.dart';
+
+void main() => runApp(const SampleApp());
```

Comment:
I wish we'd reserve `=>` for one-liners that return a value.

Reply:
But it _does_, it returns `void`! :-) (I can tell you're unpersuaded: switched to brackets)